jaxb/src/main/java/com/fasterxml/jackson/module/jaxb/JaxbAnnotationIntrospector.java

```diff
@@ -12,6 +12,7 @@
 import com.fasterxml.jackson.annotation.JsonAutoDetect.Visibility;
 import com.fasterxml.jackson.annotation.*;
 import com.fasterxml.jackson.core.*;
+import com.fasterxml.jackson.core.json.PackageVersion;
```

Comment:
Ah, no, that would bring wrong `PackageVersion` -- probably brought in by IDE accidental. This is auto-generated from template as part of build process.